x-pack/test/apm_api_integration/tests/mobile/mobile_terms_by_field.spec.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { APIReturnType } from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
+import { ENVIRONMENT_ALL } from '@kbn/apm-plugin/common/environment_filter_values';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { generateMobileData } from './generate_mobile_data';
+
+type MobileTermsByField = APIReturnType<'GET /internal/apm/mobile-services/{serviceName}/terms'>;
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const apmApiClient = getService('apmApiClient');
+  const registry = getService('registry');
+  const synthtraceEsClient = getService('synthtraceEsClient');
+
+  const start = new Date('2023-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2023-01-01T00:15:00.000Z').getTime() - 1;
+
+  async function getMobileTermsByField({
+    environment = ENVIRONMENT_ALL.value,
+    kuery = '',
+    serviceName,
+    fieldName,
+    size = 10,
+  }: {
+    environment?: string;
+    kuery?: string;
+    serviceName: string;
+    transactionType?: string;
+    fieldName?: string;
```

Comment:
Isn't `fieldName` required?

Reply:
No, it is https://github.com/elastic/kibana/pull/152604/files#diff-1ad453ab691a31f8618235faaa7cdd5c0f0f329635a5fb68ffbe358f2a4d4b59R35 

I accidentally left it optional in the tests. will fix it